package.json

```diff
@@ -17,8 +17,8 @@
   "type": "module",
   "engineStrict": true,
```

Comment:
hey, would you like to have `engineStrict` everywhere? seems like it can't hurt and is related to this PR.